core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -1037,237 +923,6 @@ class DynamicBrokerReconfigurationTest extends QuorumTestHarness with SaslSetup
     stopAndVerifyProduceConsume(producerThread, consumerThread)
   }
 
-  @Test
-  // Modifying advertised listeners is not supported in KRaft
```

Comment:
Out of curiosity, do we have plans to support this for KRaft? I saw there's a JIRA about revising the docs (https://issues.apache.org/jira/browse/KAFKA-16617)